### ISSUE TITLE
ListItem issue passing A component

### ISSUE THREAD

**Issue by dkdewitt** [mxstbr/react-boilerplate#365]
Issue or question on the proper use of A components in a List component.

```
  render() {

    const navItems = 
      [
        {route: "/", text: "Home"},
        {route: "/", text: "About"},
      ];

    let items = navItems.map((item) => {
      return (
        <A href={item.route}>{item.text}</A> 
        );
    });
    return (
      <div>
        <List items={items} component={ListItem} />
      </div>
    );  
  }
```

does not render properly unless I change the List component:
```<ComponentToRender key={`item-${index}`} item={item} />```
to
```<ComponentToRender key={`item-${index}`} content={item} />```

Am I using this correctly?  Seems this should work the way I currently have it.

**Comment by ralomucho** [comment]
RepoListItem expects an 'item' props whereas Listitem expects a 'content' props. Two different names for the same thing.
Gonna submit a PR so that you don't need to  change List code

**Comment by mxstbr** [comment]
Fixed in #372! 

We generally don't recommend using the components of the example application for real. They aren't the best components ever made, and are more to showcase the general concepts used in this boilerplate!
